### ISSUE TITLE
Render VJ Include on AMP in fixed size iframe

### ISSUE THREAD

**Issue by andrewscfc** [bbc/simorgh#6917]
**Is your feature request related to a problem? Please describe.**
We need to be able to render VJ includes on AMP pages.

**Describe the solution you'd like**
- We need render an `amp-iframe` to bring VJ Includes into our AMP pages. This can construct an amp iframe like this: `<amp-iframe src="https://news.files.bbci.co.uk/include/vjamericas/176-eclipse-lookup/mundo/app/amp"></amp-iframe>`
- The iframe should be centred to be consistent with social embeds, this will be changed in a subsquent issue (see Jira NEWSART-943 - TO DO @andrewscfc )
- The amp-iframe must not exceed the width of our main content
- The amp-iframe must have the most appropriate attributes for accessiblity
- Pair with UX during development to capture problems early
- This iframe will [use a placeholder](https://amp.dev/documentation/components/amp-iframe/#iframe-with-placeholder) using the url provided in the include url
  - If no placeholder image is provided in the url we do not render the include at all

**Placeholder**
- The placeholder default size should be 640px (wide) by 360px (height).
- Where provided:
  - we will set width of the iframe to the value of `amp-image-width` from the include url
  - we will set height of the iframe to the value of `amp-image-height` from the include url

Example include url with parameters mentioned above: https://www.bbc.com/ws/includes/include/newsspec/26926-covid-data-table/mundo/app?responsive=true&newsapps=true&app-image=https://news.files.bbci.co.uk/vj/live/idt-images/image-slider-asdf/app_launcher_cv_table_ws_640_3x-nc_v0mmu.png&app-clickable=true&amp-clickable=true&amp-image-height=360&amp-image-width=640&amp-image=https://news.files.bbci.co.uk/vj/live/idt-images/image-slider-asdf/app_launcher_cv_table_ws_640_3x-nc_v0mmu.png?v=103 

**A11y requirements**
We should ensure the amp iframe has the most appropriate attributes for accessiblity as part of this issue.

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc
- Ensure we have the necessary integration test coverage for this case.

- [x] This feature is expected to need manual testing.

**Additional context**
https://www.bbc.com/mundo/amp/noticias-51705060

```
<amp-iframe width="640" height="723" sandbox="allow-scripts allow-same-origin allow-top-navigation-by-user-activation allow-forms" layout="responsive" frameborder="0" resizable="" src="https://news.files.bbci.co.uk/include/newsspec/26866-coronavirus-mapped/mundo/app/amp" class="i-amphtml-element i-amphtml-layout-responsive i-amphtml-layout-size-defined i-amphtml-layout" i-amphtml-layout="responsive" scrolling="no" style="--loader-delay-offset:235ms !important; height: 723px;"></amp-iframe>
```

![image](https://user-images.githubusercontent.com/9645462/84164575-95850700-aa6a-11ea-9654-75104c74173a.png)

**Alephant - legacy AMP renderer**
```
<div class="article_include__vj_content amp-u-mt+ amp-u-mb+">
  {{#interactive_content?}}
    <amp-iframe width={{ include_placeholder_width }} height={{ include_placeholder_height }}
        sandbox="allow-scripts allow-same-origin allow-top-navigation-by-user-activation allow-forms"
        layout="responsive"
        frameborder="0"
        {{# show_more_translation }}
            resizable
        {{/ show_more_translation }}
        src={{ include_iframe_src }}>
        {{# include_placeholder_path }}
          <amp-img width={{ include_placeholder_width }} height={{ include_placeholder_height }} layout="fill" src={{ include_placeholder_path }} placeholder></amp-img>
        {{/ include_placeholder_path }}
        {{# show_more_translation }}
            <div tabindex="0" role="button" aria-label="{{ show_more_translation }}" class="amp-show-more__button" overflow>
                <span>
                    {{ show_more_translation }}
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="32px"
                        height="30px" viewBox="0 0 32 32" enable-background="new 0 0 32 32" xml:space="preserve" class="amp-show-more__icon">
                        <g id="Grid"></g>
                        <g id="Layer_2">
                            <polygon points="16,29 32,3 24.8,3 16,18.3 7.2,3 0,3"/>
                        </g>
                    </svg>
                </span>
            </div>
        {{/ show_more_translation }}
    </amp-iframe>
  {{/interactive_content?}}
  {{^interactive_content?}}
    <amp-img width={{ include_placeholder_width }} height={{ include_placeholder_height }}  layout="responsive" src={{ include_placeholder_path }}></amp-img>
  {{/interactive_content?}}
</div>
```

**Comment by simonsinclair** [comment]
This is blocked pending https://github.com/bbc/simorgh/issues/5989.
